.appveyor.yml

```diff
@@ -10,8 +10,8 @@ cache:
 environment:
     global:
         CONDA_CHANNELS: conda-forge
-        CONDA_DEPENDENCIES: pip setuptools wheel cython mock six biopython networkx joblib matplotlib scipy vs2015_runtime pytest mmtf-python GridDataFormats hypothesis pytest-cov codecov chemfiles
-        PIP_DEPENDENCIES: gsd==1.9.3 duecredit parmed
+        CONDA_DEPENDENCIES: pip setuptools wheel cython mock six biopython networkx joblib matplotlib scipy vs2015_runtime pytest mmtf-python GridDataFormats hypothesis pytest-cov codecov
+        PIP_DEPENDENCIES: gsd==1.9.3 duecredit parmed tqdm
```

Comment:
Is there no tqdm conda package for Windows? (Generally we prefer the conda package.)

Reply:
Yes there is, good to know. I'll change that